docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -2,88 +2,180 @@
+
+   * `uuidsEqual()`
+      ```
+      var first = tizen.bluetooth.uuidsEqual("1234", "00001234");
+      var second = tizen.bluetooth.uuidsEqual("ab5690ef", "ab5690ef-0000-1000-8000-00805F9B34FB");
+      var third = tizen.bluetooth.uuidsEqual("abcdef01-2345-6789-abcd-ef0123456789", "abcdef01");
+      ```
+
+      Both `first` and `second` are `true`. `third` is false.
+
+### Handling binary data
+The `Bytes` type, that aggregates all types in Bluetooth API used to pass binary data. It can be either a *byte[]* or a *DOMString* or a *Uint8Array*.
+
+`BluetoothManager` methods to manipulate binary data:
+   * `toByteArray()`
+      ```
+      var dataInt8Array = new Int8Array([24, 177]);
+      var dataUint8Array = new Uint8Array([24, 177]);
+      var dataString = "0x18b1";
+
+      var first = tizen.bluetooth.toByteArray(dataInt8Array);
+      var second = tizen.bluetooth.toByteArray(dataUint8Array);
+      var third = tizen.bluetooth.toByteArray(dataString);
+      ```
+      `first`, `second` and `third` variables are equal arrays.
+
+   * `toDOMString()`
+      ```
+      var dataInt8Array = new Int8Array([24, 177]);
+      var dataUint8Array = new Uint8Array([24, 177]);
+      var dataString = "0x18b1";
+
+      var first = tizen.bluetooth.toDOMString(dataInt8Array);
+      var second = tizen.bluetooth.toDOMString(dataUint8Array);
+      var third = tizen.bluetooth.toDOMString(dataString);
+      ```
+      `first`, `second` and `third` variables are equal arrays.
+
+   * `toUint8Array()`
+      ```
+      var dataInt8Array = new Int8Array([24, 177]);
+      var dataUint8Array = new Uint8Array([24, 177]);
+      var dataString = "0x18b1";
+
+      var first = tizen.bluetooth.toUint8Array(dataInt8Array);
+      var second = tizen.bluetooth.toUint8Array(dataUint8Array);
+      var third = tizen.bluetooth.toUint8Array(dataString);
+      ```
+      `first`, `second` and `third` variables are equal.
+
+
+
+## The main features of the Bluetooth API include:
+
+- Managing the local Bluetooth adapter
 
   You can [enable and disable the local Bluetooth adapter](#managing-the-local-bluetooth-adapter), and change the device name for it.
 
-- Discovering devices   
+- Discovering devices
 
   You can [discover other Bluetooth devices](#discovering-bluetooth-devices).
 
-- Creating a bond with a Bluetooth device   
+- Creating a bond with a Bluetooth device
 
   You can [create a bond](#creating-a-bond-with-a-bluetooth-device) with another device retrieved through the discovery process. Bonding allows the 2 devices to establish a connection.
 
-- Connecting to and exchanging data with a Bluetooth device   
+- Connecting to and exchanging data with a Bluetooth device
 
   You can [connect to and exchange data with a remote Bluetooth device](#connecting-to-and-exchanging-data-with-a-bluetooth-device).
 
-The main Bluetooth (4.0) Low Energy features include:
+## The main Bluetooth (4.0) Low Energy features include:
```

Comment:
Please avoid using colons in heading. Suggestion

## Main features of Bluetooth (4.0) Low Energy

The main features of the  Bluetooth (4.0) Low Energy features include:

Reply:
I have applied your suggestions.